Remove calls to fdatasync (closes: #559686). 
Upstream is going to do the same in next release.
Index: sysstat/sadc.c
===================================================================
--- sysstat.orig/sadc.c	2011-01-19 21:56:42.000000000 +0100
+++ sysstat/sadc.c	2011-01-19 21:56:57.000000000 +0100
@@ -903,11 +903,13 @@
 			 */
 			do_sa_rotat = FALSE;
 
+#if 0 /* Debian bug #559686 */
 			if (fdatasync(ofd) < 0) {
 				/* Flush previous file */
 				perror("fdatasync");
 				exit(4);
 			}
+#endif
 			close(ofd);
 			strcpy(ofile, new_ofile);
 
